src/main/resources/org/jenkinsci/plugins/workflow/support/steps/build/RunWrapper/help.html

```diff
@@ -1,4 +1,6 @@
 <dl>
+    <dt><code>getBuildCauses</code></dt><dd>Returns a JSON array of build causes for the current build</dd>
+    <dt><b>EXPERIMENTAL - MAY CHANGE</b>  <code>getBuildCauses(String causeClass)</code></dt><dd>Takes a string representing the fully qualified <code>Cause</code> class and returns a <code>JSON array</code> of build causes filtered by that type for the current build, or an empty <code>JSON array</code> if no causes of the specified type apply to the current build</code></dd>
```

Comment:
Sounds a lot like the Beta annotation

Reply:
Is there a preferred wording @svanoort ?  I did some quick searches but couldn't find anything that  looked like a pattern.

Reply:
No, no, I mean on the API itself in the Java code -- but it's not surfaced for users in such a way (more of a purely-developer-side thing).

Reply:
Anyway, no chance needed to the code here

Reply:
oh yeah, i saw that but felt for this API it wasn't needed as this will almost surely only be called from pipeline scripts